lib/v8debugapi.js

```diff
@@ -277,6 +280,12 @@ module.exports.create = function(logger_, config_) {
     // TODO: need test coverage for the above
     // TODO: work with debug server to come up with a more robust solution
 
+    if (breakpoint.location.line >= matchingScript.line_ends.length) {
+      return setErrorStatusAndCallback(cb, breakpoint,
+        StatusMessage.BREAKPOINT_SOURCE_LOCATION,
+        messages.INVALID_LINE_NUMBER);
```

Comment:
Ahh, I see that now. [The Google one](https://google.github.io/styleguide/javascriptguide.xml) seems like a good place to start but I am not yet familiar with it.